doc/CONFIGURE.md

```diff
@@ -0,0 +1,205 @@
+
+Finally the agent should have **permissions to read and write on the work directory**, and **enough disk space** to store *remoting.jar* (~1MB),
+logs (~10MB should be enough), and your build workspaces (it depends of your builds, probably a few GB).
+
+# Configure a Node to use SSH Slaves plugin
+
+## Create a Node
+First of all, we have to create a node, you have to go to `Manage Jenkins/Manage Nodes`, 
+
+![](images/new-node-button.png)
+
+then click on the `New Node` button and configure the name and Node type.
+
+![](images/new-node-config.png)
+
+After creating the new node, you have to configure the node settings
+
+* **Name:** Name that uniquely identifies an agent within this Jenkins installation.
+* **Description:** Optional human-readable description for this agent. 
+* **\# of Executors:** The maximum number of concurrent builds that Jenkins may perform on this agent.
+* **Remote root directory:** An agent needs to have a directory dedicated to Jenkins. 
+Specify the path to this directory on the agent. It is best to use an absolute path, 
+such as /var/jenkins or c:\jenkins. This should be a path local to the agent machine.
+* **Labels:** Labels (or tags) are used to group multiple agents into one logical group. 
+You will use them to select it as agent for a build. Multiple labels must be separated by a space.
+* **Usage:** Controls how Jenkins schedules builds on this node.
+    * **Use this node as much as possible:** This is the default setting. In this mode, Jenkins uses this node freely.
+    Whenever there is a build that can be done by using this node, Jenkins will use it.
+    * **Only build jobs with label expressions matching this node:**
+    In this mode, Jenkins will only build a project on this node when that project is restricted to certain nodes using a label expression,
+    and that expression matches this node's name and/or labels.
+    This allows a node to be reserved for certain kinds of jobs. For example, if you have jobs that run performance tests,
+    you may want them to only run on a specially configured machine, while preventing all other jobs from using that machine.
+    To do so, you would restrict where the test jobs may run by giving them a label expression matching that machine. 
+    Furthermore, if you set the # of executors value to 1, you can ensure that only one performance test will execute at 
+    any given time on that machine; no other builds will interfere.
+* **Launch method:** Select *Launch agent agents via SSH* Starts a agent by sending commands over a secure SSH connection.
+
+![](images/ssh-node-basic-config.png)
+
+## Configure Launch agent agents via SSH
+
+Once you selected the **Launch method** to **Launch agent agents via SSH**, you can configure your SSH agent settings.
+
+### Required settings
+
+* **Host:** Hostname or IP of the agent, it should be resolved and reachable from the Jenkins instance. 
```

Comment:
Probably best to change "resolved and reachable" to "resolvable and reachable". In other words, they should use a matching form.